### PR TITLE
Update traditional-installer.md

### PR DESCRIPTION
**Pull request by CrossZX**

### CODE REVIEW

File: ProGet/installation/installation-guide/traditional-installer.md [hunk 1]
```diff
@@ -1,108 +1,175 @@
 ---
-title: Traditional Installer Guide
+title: Traditional (Legacy) Installer Guide
 sequence: 500
-keywords: proget, installation
+keywords: proget, installation, legacy, traditional, silent install
```
Comment (apxltd) [inline ProGet/installation/installation-guide/traditional-installer.md:4]: no keywords

File: ProGet/installation/installation-guide/traditional-installer.md [hunk 2]
```diff
@@ -1,108 +1,175 @@
...
-If any of the items indicate an error, you can click on it to go back to that page and fix it.
-
-## 13. Patiently Wait {#patiently-wait data-title="Patiently Wait"}
-
-If you chose to have SQL Server Express installed, then the actual installation process will take a while, as the installer must download the SQL installer and then wait for it to complete. On some servers, we've seen this take as long as fifteen minutes.
-
-![](/resources/documentation/proget-installation/progetclickinstall.png){.screenshot}
-
-Clicking cancel (especially during the SQL Express installation) is not advised, as you may have to manually remove some bits of SQL Server using Add/Remove programs. Once everything's done, you'll be ready to start using ProGet.
+*We are planning to stop shipping legacy installer in 2021*
```
Comment (apxltd) [inline ProGet/installation/installation-guide/traditional-installer.md:11]: Move to attn block above, no need for italics.

File: ProGet/installation/installation-guide/traditional-installer.md [hunk 3]
```diff
@@ -1,108 +1,175 @@
...
+
+For the most convenient way to install ProGet is to use the [InedoHub](https://my.inedo.com/downloads). It is primarily used to install, update, and otherwise service applications in the Inedo Suit. You can learn more about it on how the InedoHub can help you save time by reading our [guide](https://docs.inedo.com/docs/desktophub/overview).
```
Comment (apxltd) [inline ProGet/installation/installation-guide/traditional-installer.md:13]: * first link to Inedo Hub should be the overview docs (i.e. ttps://docs.inedo.com/docs/desktophub/overview)
* fix typos
* Inedo Hub (has a space)
* reading our [guide] --> [reading our guide]

File: ProGet/installation/installation-guide/traditional-installer.md [hunk 4]
```diff
@@ -1,108 +1,175 @@
...
+
+Please also note that if you have the InedoHub installed you will NOT be able to use the traditional install method without first completely removing the InedoHub and any products installed using it.
```
Comment (apxltd) [inline ProGet/installation/installation-guide/traditional-installer.md:15]: Note that if you have the Inedo Hub installed you will **not** be able to use the traditional installer without first [uninstalling the Inedo Hub](/docs/desktophub/overview#uninstalling) and any products installed using it.

File: ProGet/installation/installation-guide/traditional-installer.md [hunk 5]
```diff
@@ -1,108 +1,175 @@
...
+
+If you require a legacy installer for any version of ProGet, you can on the [ProGet All Version](https://my.inedo.com/proget/versions) page, and clicking on the “Traditional Installer” for the version you want.
```
Comment (apxltd) [inline ProGet/installation/installation-guide/traditional-installer.md:17]: * fix typo
* legacy installer -> traditional (legacy) installer

File: ProGet/installation/installation-guide/traditional-installer.md [hunk 6]
```diff
@@ -1,108 +1,175 @@
...
+
+Once you have downloaded and opened the installer, you’ll be asked to review and accept out [license agreement](https://inedo.com/proget/license-agreement) before you can use ProGet
```
Comment (apxltd) [inline ProGet/installation/installation-guide/traditional-installer.md:19]: * open -> run
* typo

File: ProGet/installation/installation-guide/traditional-installer.md [hunk 7]
```diff
@@ -1,108 +1,175 @@
...
+
+Complete the rest of the installation by selecting each setting appropriate for your system. We have updated our [detailed installation guide](https://docs.inedo.com/docs/proget/installation/installation-guide) to reflect the new streamlined layout of the InedoHub, but the options are still the same.
```
Comment (apxltd) [inline ProGet/installation/installation-guide/traditional-installer.md:21]: Although the [ProGet installation guide](/docs/proget/installation/installation-guide) uses the Inedo Hub, the settings are mostly the same.

**Review by apxltd — CHANGES_REQUESTED** [review]

**Review by CrossZX — COMMENTED** [review]
Updated as per feedback received

**Pull request merged**